cron/internal/data/iterator.go

```diff
@@ -47,10 +52,29 @@ type csvIterator struct {
 	decoder *csvutil.Decoder
 	err     error
 	next    RepoFormat
+	count   int
+}
+
+// check if the most recently decoded record is a header, and skip it if it is.
+func (reader *csvIterator) ignoreHeader() {
+	header, err := csvutil.Header(RepoFormat{}, "csv")
+	if err != nil {
+		reader.err = err
+		return
+	}
+	lastRead := reader.decoder.Record()
+	if reflect.DeepEqual(header, lastRead) {
+		reader.err = reader.decoder.Decode(&reader.next)
+	}
 }
 
 func (reader *csvIterator) HasNext() bool {
 	reader.err = reader.decoder.Decode(&reader.next)
+	// only compares the first read against the csv header for performance
+	if reader.count == 0 {
+		reader.ignoreHeader()
+	}
+	reader.count++
```

Comment:
`reader.count` can simply be a `bool` which is updated inside `ignoreIfHeader` fn?